src/api/app/controllers/webui/subscriptions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Webui::SubscriptionsController < Webui::WebuiController
+  before_action :require_admin
+
+  def index
+    @subscriptions_form = subscriptions_form
+  end
+
+  def update
+    subscriptions_form.update!(params[:subscriptions])
```

Comment:
First and foremost we want our users succeed in what they are trying to do. If something goes wrong storing the `EventSubscription` we want to help them resolve this. Throwing `ActiveRecord::RecordInvalid` (the 500 page) doesn't. Telling them `EventSubscription` failed because blah blah blah` does.